src/lib/common/StreetViewController.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+  import type { MapMouseEvent } from "maplibre-gl";
+  import { map, userSettings } from "../../stores";
+  import { Radio } from "../govuk";
+  import DefaultButton from "../govuk/DefaultButton.svelte";
+  import SecondaryButton from "../govuk/SecondaryButton.svelte";
+  import { getRoadLayerHelpers, type LayerHelper } from "../maplibre";
+  import CollapsibleCard from "./CollapsibleCard.svelte";
+
+  export let displayEnableButton: boolean = false;
+  export let escapeKeyExits: boolean = false;
+  const roadLayerHelpers: Array<LayerHelper> = getRoadLayerHelpers();
+  let isActive: boolean;
```

Comment:
The functionality is working great! Only big problem is when we have a browse layer turned on. Then a click will have two effects. I think we can fix this by:

1) Creating a store similar to `isAToolInUse` called `interactiveLayersEnabled` or something, defaulting to true
2) Using that in `lib/common/InteractiveLayer.svelte`. For the 3 mouse event handlers there, immediately check this store, and bail out when appropriate.
3) export `isActive` from StreetViewController.svelte and in LayerControls, bind it to the store